src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,31 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: serif;
+}
```

Comment:
- [ ] (Required) When you use CSS with React, the styling of a component can affect the styling of other components even though you don't import the CSS style. I guess that is the reason why most developers use CSS modules when styling. You reset the `font-family` to `serif` instead of `Lato` and so all other components use the `serif` font. I would advise you to use `Lato` instead of `serif`. Remember that one of the major requirements is for you to follow the design of the template given and also to use the same font face, size and weight. 👍